finalProject.html

```diff
@@ -45,25 +46,26 @@ <h2>Shopping-Cart</h2>
             <p id="price">$24.99 </p>
         </div>
         <div class="item3">
-            <h3>Quantity</h3>
-            <button type="button" onclick="addToCart(19.99)">Add</button><button type="button"
+            <p>Quantity</p>
+            <button type="button" onclick="addToCart(19.99)" id="add-tie">Add Tie</button><button type="button"
```

Comment:
You don't need to specify that the button is a button, you can remove `type="button"` on all of these.

Reply:
Easy day. All type="button" will be removed.